### PR TITLE
chore: make sure pnpm7 is used for this repo

### PR DESCRIPTION
**Pull request by dominikg**
using pnpm6 and 7 in parallel would cause problems with lockfile format changes and incompatibilities in package script syntax.

packageManager field allows people to use corepack if they like and the field is evaluated by some tools like `@antfu/ni`. renovate bot also needs a hint, otherwise it is using pnpm7 and it's PRs may fail.

Once we decide to update to 7 we just bump the versions in package.json and everyone using the repo gets notified on next use (or corepack handles it automatically).

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: pnpm/action-setup@v2.2.1
         with:
-          version: 6.23.2
+          version: 6.32.11
```
Comment (bluwy) [inline .github/workflows/ci.yml:20]: We can also omit this as `pnpm/action-setup` reads `packageManager` too.

Comment (dominikg) [reply .github/workflows/ci.yml:20]: https://github.com/pnpm/action-setup/blob/57b9359b4c69c79ca51422d02ca739a7957f4aed/src/install-pnpm/run.ts#L25-L27 

this is not great. basically curlbashing a hardcoded version of pnpm to then install another version? I don't get it.

what about this?
read packageManager from package.json and send that to  `npm i -g`
```bash
PNPM_VER=$(jq -r '.packageManager | if .[0:5] == "pnpm@" then .[5:] else "packageManager in package.json does not start with pnpm@\n" | halt_error(1)  end' package.json)
npm i -g pnpm@$PNPM_VER
```

that would exit with 1 if packageManager isn't set or doesn't start with pnpm@

**Review by benmccann — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (benmccann) [reply .github/workflows/ci.yml]: I removed the version for now and will see if we can get the issue fixed in `action-setup`: https://github.com/pnpm/action-setup/issues/38

File: packages/kit/test/utils.js [hunk 2]
```diff
@@ -162,7 +162,8 @@ export const config = {
 	],
 	use: {
 		screenshot: 'only-on-failure',
-		trace: 'retain-on-failure'
+		trace: 'retain-on-failure',
+		channel: 'chrome'
```
Comment (benmccann) [inline packages/kit/test/utils.js:166]: should we add a comment here explaining why we added this?

File: package.json [hunk 3]
```diff
@@ -40,5 +40,9 @@
 		"turbo": "^1.2.6",
 		"typescript": "~4.6.2"
 	},
+	"packageManager": "pnpm@7.1.0",
+	"engines": {
+		"pnpm": "^7.1.0"
```
Comment (benmccann) [inline package.json:45]:
```suggestion
		"pnpm": "^7.0.0"
```

Comment (benmccann) [inline package.json:45]:
```suggestion
		"pnpm": "^7.0.0"
```

**Pull request merged**